swap/src/monero/wallet.rs

```diff
@@ -26,33 +26,40 @@ pub struct Wallet {
 
 impl Wallet {
     /// Connect to a wallet RPC and load the given wallet by name.
-    pub async fn open_or_create(url: Url, name: String, env_config: Config) -> Result<Self> {
```

Comment:
I think the previous name was better. It should be made explicit that we'll be creating a wallet if it does not exist.